### PR TITLE
V2 docs

### PR DESCRIPTION
**Pull request by Quramy**
### What I did

- Rewrite README
- Write how to migration from SCS v1 or zisui
- add script to generate TOC for each *.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,301 +3,284 @@
...
+```ts
+type Variants = {
+  [variantName: string]: {
+    extends?: string | string[]; // default: ""
+    delay?: number;
+    waitImages?: boolean;
+    waitFor?: string | () => Promise<void>;
+    fullPage?: boolean;
+    hover?: string;
+    focus?: string;
```
Comment (tsuyoshiwada) [inline README.md]: I would like to add the following functions. I think we can deal with the uniformity of the API and wider user needs :dog:

- `:visited`
- `:active`

File: README.md [hunk 2]
```diff
@@ -3,301 +3,284 @@
...
+      },
+      small: {
+        width: 375,
+        height: 668,
+      },
+      xsmall: {
+        width: 320,
+        height: 568,
+      },
+    },
```
Comment (tsuyoshiwada) [inline README.md]: :heart:

File: README.md [hunk 3]
```diff
@@ -393,66 +436,62 @@ For example, the following setting makes the screenshot function wait for firing
...
-$ npm run test:e2e
+The following tasks remain. Contributes are welcome :smiley:
 
-# Run unit tests in watch mode
-$ npm run test:watch
-```
+- [x] Upgrade v2
+- [ ] More unit testing.
+- [ ] Extract crawler as a NPM package.
+- [ ] Capture with JS/CSS coverages.
```
Comment (tsuyoshiwada) [inline README.md]: :+1:

File: scripts/add-toc.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
```
Comment (tsuyoshiwada) [inline scripts/add-toc.js]: The above line is not necessary if the file path is passed to the `node` argument :+1:

File: README.md (hunk 1 above)
Comment (Quramy) [reply README.md]: Absolutely I agree. However, Puppeteer does not expose function to change an element's status to `:visited` nor `:active` . And I don't  know how to do this using `elementHandler` and DOM API 😭 ... Do you have any idea to implement this ?

Comment (tsuyoshiwada) [reply README.md]: It seems hard to deal with, I don't have any idea of implementation... :innocent:
(Chrome Developer Tools has functions that can be applied pseudo-classes, It's a pity that it seems to be possible...)

**Pull request merged**